src/app/common/hooks/analytics/use-analytics.ts

```diff
@@ -32,48 +35,93 @@ export function useInitalizeAnalytics() {
   }, []);
 }
 
+function getStateProps() {
+  const state = store.getState();
+  const currentNetwork = selectCurrentNetwork(state);
+  const walletType = selectWalletType(state);
+
+  return {
+    walletType,
+    currentNetwork,
+  };
+}
+
+interface AnalyticsObject {
```

Comment:
```suggestion
interface Analytics {
```

object seems redundant to me